File: Python/Estrutura Repetitiva/Enquanto/Quadrante.py
x: int; y: int

print(f"Digite os valores das coordenadas X e Y:")
x = int(input())
y = int(input())

while  (x != 0) and (y != 0):
        if (x > 0) and (y > 0):
            print(f"QUADRANTE Q1")
        elif (x < 0) and (y > 0):
            print(f"QUADRANTE Q2")
        elif (x < 0) and (y < 0):
         print(f"QUADRANTE Q3")
        elif (x > 0) and (y < 0):
         print(f"QUADRANTE Q4")


        print(f"Digite os valores das coordenadas X e Y:")
        x = int(input())
        y = int(input())
    

File: C/Estrutura Repetitiva/Enquanto/Quadrante.c
#include <stdio.h>
#include <string.h>



int main ()
{

    int x, y;

    printf ("Digite os valores das coordenadas X e Y:\n");
    scanf ("%d", &x);
    scanf ("%d", &y);


    while (x != 0 && y != 0)
    {
        if (x > 0 && y > 0 )
        {
            printf ("QUADRANTE Q1\n");
        }
        else if (x < 0 && y > 0 )
        {
            printf ("QUADRANTE Q2\n");
        }
        else if (x < 0 && y < 0 )
        {
            printf ("QUADRANTE Q3\n");
        }
        else if (x > 0 && y < 0 )
        {
            printf ("QUADRANTE Q4\n");
        }

        printf ("Digite os valores das coordenadas X e Y:\n");
        scanf ("%d", &x);
        scanf ("%d", &y);
    }


    return 0;
}